cms/actionbar.lib.php

```diff
@@ -24,19 +24,19 @@
  * @return $actionbar The list of permitted actions for the 'user' of 'page'. 
  */
 function getActionbarPage($userId, $pageId) {
-
+    global $pdb;
```

Comment:
Hi nean, 
  From now on we will following strict standards. This includes using 4 spaces instead of tab spacing.